<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SubMenu</class>
 <widget class="QWidget" name="SubMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>886</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>230</y>
     <width>451</width>
     <height>46</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="insert_label">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="insert_line"/>
    </item>
    <item>
     <widget class="QPushButton" name="confirm_ptn">
      <property name="text">
       <string>确认</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="back_ptn">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>450</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
